src/store/actions/async/index.js

```diff
@@ -19,3 +19,33 @@ const fetchHomePage = (url) => {
 // In this case, something like dispatch(fetchHomePageAction('http://ah-backend-summer-staging.herokuapp.com/'))
 // It uses redux thunk underneath and will dispatch the appropriate actions when the call returns
 export const fetchHomePageAction = createActionThunk(FETCH_HOME_PAGE_ACTION, fetchHomePage);
+
+export function getToken(localStorage) {
+    const token = localStorage.getItem('token');
+    if (!token || token.length < 1) {
+        throw "Not logged in";
+    }
+    return token;
+}
+
+export function getLoggedInUser(localStorage) {
+    const username = localStorage.getItem('username');
+    if (!username || username.length < 1) {
+        return "";
+    }
+    return username;
+}
+
+export function isLoggedIn(localStorage) {
```

Comment:
i see in some places you use arrow functions and others you choose to use es5. could we please stick to one, if that's the case

Reply:
Thanks for this @pbkabali. The bookmark feature and view bookmarks feature are different. The second PR implements this functionality.

Reply:
Oh okay. thanks for the clarification @Karuhanga.